bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -51,7 +52,7 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
      * @param  {String} indId    indicator id
      * @param  {Object} elements elements
      */
-    indicatorSelected: function ( datasrc, indId, elements ) {
+    indicatorSelected: function ( datasrc, indId, regionsets, elements ) {
```

Comment:
As there's a bunch of code passing around regionsets this variable could be named with something like regionsetRestriction or something like that. As it's used to filter out available regionsets to just include these ones, right?